src/components/customers/CustomerSettings.tsx

```diff
@@ -698,25 +709,35 @@ export const CustomerSettings = ({ customerId }: CustomerSettingsProps) => {
                   }
                 />
 
-                <Typography variant="body" color="grey700">
-                  {typeof customer?.netPaymentTerm !== 'number'
-                    ? translate(
-                        'text_64c7a89b6c67eb6c98898241',
-                        {
-                          days: billingEntity?.netPaymentTerm,
-                        },
-                        billingEntity?.netPaymentTerm,
-                      )
-                    : customer?.netPaymentTerm === 0
-                      ? translate('text_64c7a89b6c67eb6c98898125')
-                      : translate(
-                          'text_64c7a89b6c67eb6c9889815f',
-                          {
-                            days: customer?.netPaymentTerm,
-                          },
-                          customer?.netPaymentTerm,
-                        )}
-                </Typography>
+                {typeof customer?.netPaymentTerm !== 'number' && (
+                  <Typography variant="body" color="grey700">
+                    {translate(
+                      'text_64c7a89b6c67eb6c98898241',
+                      {
+                        days: billingEntity?.netPaymentTerm,
+                      },
+                      billingEntity?.netPaymentTerm,
+                    )}
+                  </Typography>
+                )}
+
+                {typeof customer?.netPaymentTerm === 'number' && customer?.netPaymentTerm === 0 && (
+                  <Typography variant="body" color="grey700">
+                    {translate('text_64c7a89b6c67eb6c98898125')}
+                  </Typography>
+                )}
+
+                {typeof customer?.netPaymentTerm === 'number' && customer?.netPaymentTerm !== 0 && (
+                  <Typography variant="body" color="grey700">
+                    {translate(
+                      'text_64c7a89b6c67eb6c9889815f',
+                      {
+                        days: customer?.netPaymentTerm,
+                      },
+                      customer?.netPaymentTerm,
+                    )}
+                  </Typography>
+                )}
```

Comment:
Very verbose. As it renders the same Typo with same attributes, can we instead have a method, placed above, that will extract the logic from the template?